teams/wg-async.toml

```diff
@@ -42,3 +42,12 @@ name = "Async working group"
 description = "Pursuing core language and library support for async-await"
 repo = "https://github.com/rust-lang/wg-async"
 zulip-stream = "wg-async"
+
+[[zulip-groups]]
+name = "WG-async"
+
+[[zulip-groups]]
+name = "WG-async-sprint-planning"
+extra-people = [
+  "hawkw"
```

Comment:
I don't think I remember @hawkw coming to meetings, so I'm not sure she wants to be in this list

Reply:
Removed for now. We can always re-add later.